### PR TITLE
Fix for page layout fix for blog posts without image.

### PR DESCRIPTION
**Pull request by bharath-kondeti**

### CODE REVIEW

File: js/gated-content/src/views/BlogPage.vue [hunk 1]
```diff
@@ -5,10 +5,13 @@
     </div>
     <div v-else-if="error">Error loading</div>
     <template v-else>
-      <div class="blog-page__image" v-bind:style="{
-              backgroundImage: `url(${image})`
-      }">
+      <div v-if="blog.attributes.field_vy_blog_image !== null">
```
Comment (froboy) [inline js/gated-content/src/views/BlogPage.vue:8]: @bharath-kondeti is it possible to put this `v-if` on the div below so we're not creating unnecessary divs?

So, something like `<div class="blog-page__image" v-if="blog.attributes.field_vy_blog_image !== null" v-bind:...`

File: js/gated-content/src/views/BlogPage.vue [hunk 2]
```diff
@@ -5,10 +5,13 @@
...
+        <div class="blog-page__image" v-bind:style="{
+                backgroundImage: `url(${image})`
+        }">
+        </div>
       </div>
+      <div v-else><div class="blog-page__no-image"></div></div>
```
Comment (froboy) [inline js/gated-content/src/views/BlogPage.vue:14]: Why is an empty div needed here? I tested on the sandbox and `div.blog-page__image` being gone seems sufficient to fix the formatting.

**Review by froboy — CHANGES_REQUESTED** [review]
@bharath-kondeti a few questions/comments here. Also, please run `npm run build` and commit the resulting files into this branch.

**Comment by froboy** [comment]
@bharath-kondeti thanks for the update. This is much cleaner now. Either I or one of the Open Y team will need to test, merge in the latest, and then regenerate the compiled `js`.

**Comment by anpolimus** [comment]
@bharath-kondeti may I ask you to rebuild app against the last master?
Code is ok but we have to fix merge conflict first.

**Comment by bharath-kondeti** [comment]
@anpolimus I have worked on the merge conflicts. Please verify the same.

**Comment by anpolimus** [comment]
Ok thx

**Review by hamrant — DISMISSED** [review]

**Comment by hamrant** [comment]
@bharath-kondeti may I ask you to rebuild app one more time?

**Comment by bharath-kondeti** [comment]
@hamrant, I will do that.

File: modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/config/openy_gc_auth.provider.daxko_sso.yml [hunk 3]
```diff
@@ -1 +1 @@
-redirect_url: '/gated_content'
+redirect_url: '/openy-gc-auth-daxko-sso/back-redirect'
```
Comment (podarok) [inline modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/config/openy_gc_auth.provider.daxko_sso.yml:1]: upgrade path needed

**Review by podarok — CHANGES_REQUESTED** [review]

**Review by anpolimus — APPROVED** [review]

**Pull request merged**